src/plugin.js

```diff
@@ -168,11 +169,21 @@ const initPlugin = function(player, options) {
         if (player.ended()) {
           return resetMonitor();
         }
+
       }
 
       fn.call(this);
     };
 
+    if (videojs.browser.IS_IOS) {
```

Comment:
We should add a comment here explaining why this entire block exists as it's not going to be clear in the future.